doc/python_api/rst/bge_types/bge.types.KX_GameObject.rst

```diff
@@ -595,7 +599,7 @@ base class --- :class:`SCA_IObject`
       :return: The vector in relation to the objects rotation.
       :rtype: 3d vector.
 
-   .. method:: applyMovement(movement, local=False)
+   .. method:: applyMovement(movement[, local])
```

Comment:
hello, it seems you forgot to indicate the default local value here and for applyRotation too.